os/include/netdb.h

```diff
@@ -339,6 +350,9 @@ FAR struct hostent *gethostbyaddr(FAR const void *addr, socklen_t len, int type)
 
 /**
  * @brief gethostbyname() is the function sending DNS query to DNS server with hostname
+ *
+ * @details @b #include <netdb.h>\n
+ * SYSTEM CALL API
```

Comment:
please check if this API is SYSTEM CALL or not.

Reply:
revised